src/test/java/lotto/exception/LottoNumberExceptionTest.java

```diff
@@ -0,0 +1,67 @@
+package lotto.exception;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.*;
+
+class LottoNumberExceptionTest {
+    LottoNumberException lottoNumberException = new LottoNumberException();
+
+    @ParameterizedTest
+    @ValueSource(strings = {"", " ", "      "})
+    @DisplayName("빈칸을 입력 하면 예외가 발생 한다.")
+    void checkBlank(String input) {
+        assertThatThrownBy(() -> lottoNumberException.checkBlank(input))
+                .isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @ParameterizedTest
```

Comment:
`@ParameterizedTest` 역시 이름을 설정해 테스트 실행 결과를 조금 더 가독성이 좋게 만들 수 있습니다!
```suggestion
    @ParameterizedTest(name = "{0}을 입력하면 예외가 발생한다.")
```
![image](https://github.com/woowacourse-precourse/java-lotto-6/assets/76612738/61de6187-91b6-4927-b731-3b06c7f8d1c0)

Reply:
아하..! 이런 방법은 처음 알게되었네요!
사실 테스트하다보면 그냥 success가 떴으니 맞겠지 하고 넘어갔는데
이렇게 보니 터미널만 봐도 어떤 데이터를 테스트했는지 보기 좋네요
좋은 의견 감사합니다!!